tests/phpunit/tests/kses.php

```diff
@@ -2188,4 +2188,98 @@ public function data_kses_globals_are_defined() {
 
 		return $this->text_array_to_dataprovider( $required_kses_globals );
 	}
+
+	/**
+	 * @ticket 29807
+	 */
+	function test_wp_filter_post_kses_img() {
+	global $allowedposttags;
+
+		$attributes = array(
+			'class' => 'classname',
+			'id' => 'idattr',
+			'style' => 'color: red;',
+			'alt' => 'alt',
+			'src' => '/test.png',
+			'srcset' => '/test.png 1x, /test-2x.png 2x',
```

Comment:
Seems it may be good to add one more URI, perhaps `,/test-3x.png 3x` without a space between the comma the the URI (this is still a valid HTML syntax afaik)?